language/move-compiler/src/shared/mod.rs

```diff
@@ -555,6 +556,12 @@ pub mod known_attributes {
         ExpectedFailure,
     }
 
+    #[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
+    pub enum NativeAttribute {
+        // It is a fake native function that actually compiles to a bytecode instruction
```

Comment:
The term "fake" is used because the design is the functions won't exist in the compiled module. 
Normally, a `native` function still produces an entry in function definitions, just with no code. 

For this PR, they are still in the compiled module. After the stackless bytecode gets full support, we will add the filtering.